08:06:14.761 [http-nio-8080-exec-10] ERROR r.s.l.s.w.service.EmployeeService - There is no report by id 1 
08:06:39.964 [http-nio-8080-exec-1] ERROR r.s.l.s.w.service.EmployeeService - There is no report by id 2 
08:07:41.357 [http-nio-8080-exec-4] ERROR r.s.l.s.w.service.EmployeeService - There is no report by id 6 
08:41:46.645 [http-nio-8080-exec-3] ERROR r.s.l.s.w.service.EmployeeService - Неверный запрос пользователя
06:59:30.048 [main] ERROR o.s.boot.SpringApplication - Application run failed
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'liquibase' defined in class path resource [org/springframework/boot/autoconfigure/liquibase/LiquibaseAutoConfiguration$LiquibaseConfiguration.class]: Validation Failed:
     1 changesets check sum
          liquibase/changesets/02072023_create_room.yml::1::developer was: 8:5e1250fa83a20f7e5e84ed3d2154c6e3 but is now: 8:d940c51cdb7f4d5c474fe020bfa03cd2

	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1770)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:598)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:520)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:326)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:324)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:200)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:313)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:200)
	at org.springframework.context.support.AbstractApplicationContext.getBean(AbstractApplicationContext.java:1156)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:931)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:608)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:146)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:733)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:435)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:311)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1305)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1294)
	at ru.skypro.lessons.springboot.weblibraryhw.WebLibraryHwApplication.main(WebLibraryHwApplication.java:10)
Caused by: liquibase.exception.ValidationFailedException: Validation Failed:
     1 changesets check sum
          liquibase/changesets/02072023_create_room.yml::1::developer was: 8:5e1250fa83a20f7e5e84ed3d2154c6e3 but is now: 8:d940c51cdb7f4d5c474fe020bfa03cd2

	at liquibase.changelog.DatabaseChangeLog.validate(DatabaseChangeLog.java:357)
	at liquibase.Liquibase$16.run(Liquibase.java:2296)
	at liquibase.Scope.lambda$child$0(Scope.java:187)
	at liquibase.Scope.child(Scope.java:196)
	at liquibase.Scope.child(Scope.java:186)
	at liquibase.Scope.child(Scope.java:165)
	at liquibase.Liquibase.runInScope(Liquibase.java:2639)
	at liquibase.Liquibase.listUnrunChangeSets(Liquibase.java:2286)
	at liquibase.Liquibase.isUpToDateFastCheck(Liquibase.java:506)
	at liquibase.Liquibase.lambda$update$1(Liquibase.java:237)
	at liquibase.Scope.lambda$child$0(Scope.java:187)
	at liquibase.Scope.child(Scope.java:196)
	at liquibase.Scope.child(Scope.java:186)
	at liquibase.Scope.child(Scope.java:165)
	at liquibase.Liquibase.runInScope(Liquibase.java:2639)
	at liquibase.Liquibase.update(Liquibase.java:236)
	at liquibase.Liquibase.update(Liquibase.java:221)
	at liquibase.integration.spring.SpringLiquibase.performUpdate(SpringLiquibase.java:329)
	at liquibase.integration.spring.SpringLiquibase.afterPropertiesSet(SpringLiquibase.java:284)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1816)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1766)
	... 18 common frames omitted
07:01:36.092 [main] ERROR o.s.boot.SpringApplication - Application run failed
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'liquibase' defined in class path resource [org/springframework/boot/autoconfigure/liquibase/LiquibaseAutoConfiguration$LiquibaseConfiguration.class]: Validation Failed:
     1 changesets check sum
          liquibase/changesets/02072023_create_room.yml::1::developer was: 8:5e1250fa83a20f7e5e84ed3d2154c6e3 but is now: 8:d940c51cdb7f4d5c474fe020bfa03cd2

	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1770)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:598)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:520)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:326)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:324)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:200)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:313)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:200)
	at org.springframework.context.support.AbstractApplicationContext.getBean(AbstractApplicationContext.java:1156)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:931)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:608)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:146)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:733)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:435)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:311)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1305)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1294)
	at ru.skypro.lessons.springboot.weblibraryhw.WebLibraryHwApplication.main(WebLibraryHwApplication.java:10)
Caused by: liquibase.exception.ValidationFailedException: Validation Failed:
     1 changesets check sum
          liquibase/changesets/02072023_create_room.yml::1::developer was: 8:5e1250fa83a20f7e5e84ed3d2154c6e3 but is now: 8:d940c51cdb7f4d5c474fe020bfa03cd2

	at liquibase.changelog.DatabaseChangeLog.validate(DatabaseChangeLog.java:357)
	at liquibase.Liquibase$16.run(Liquibase.java:2296)
	at liquibase.Scope.lambda$child$0(Scope.java:187)
	at liquibase.Scope.child(Scope.java:196)
	at liquibase.Scope.child(Scope.java:186)
	at liquibase.Scope.child(Scope.java:165)
	at liquibase.Liquibase.runInScope(Liquibase.java:2639)
	at liquibase.Liquibase.listUnrunChangeSets(Liquibase.java:2286)
	at liquibase.Liquibase.isUpToDateFastCheck(Liquibase.java:506)
	at liquibase.Liquibase.lambda$update$1(Liquibase.java:237)
	at liquibase.Scope.lambda$child$0(Scope.java:187)
	at liquibase.Scope.child(Scope.java:196)
	at liquibase.Scope.child(Scope.java:186)
	at liquibase.Scope.child(Scope.java:165)
	at liquibase.Liquibase.runInScope(Liquibase.java:2639)
	at liquibase.Liquibase.update(Liquibase.java:236)
	at liquibase.Liquibase.update(Liquibase.java:221)
	at liquibase.integration.spring.SpringLiquibase.performUpdate(SpringLiquibase.java:329)
	at liquibase.integration.spring.SpringLiquibase.afterPropertiesSet(SpringLiquibase.java:284)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1816)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1766)
	... 18 common frames omitted
08:04:10.069 [main] ERROR liquibase.changelog - ChangeSet liquibase/changesets/02072023_create_authUser.yml::4::developer encountered an exception.
08:04:10.331 [main] ERROR o.s.boot.SpringApplication - Application run failed
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'liquibase' defined in class path resource [org/springframework/boot/autoconfigure/liquibase/LiquibaseAutoConfiguration$LiquibaseConfiguration.class]: liquibase.exception.MigrationFailedException: Migration failed for changeset liquibase/changesets/02072023_create_authUser.yml::4::developer:
     Reason: liquibase.exception.DatabaseException: ERROR: type "long" does not exist
  Позиция: 35 [Failed SQL: (0) CREATE TABLE public.auth_user (id LONG GENERATED BY DEFAULT AS IDENTITY NOT NULL, username VARCHAR(255), password VARCHAR(255), role_name VARCHAR(255), CONSTRAINT auth_user_pkey PRIMARY KEY (id))]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1770)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:598)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:520)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:326)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:324)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:200)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:313)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:200)
	at org.springframework.context.support.AbstractApplicationContext.getBean(AbstractApplicationContext.java:1156)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:931)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:608)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:146)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:733)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:435)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:311)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1305)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1294)
	at ru.skypro.lessons.springboot.weblibraryhw.WebLibraryHwApplication.main(WebLibraryHwApplication.java:10)
Caused by: liquibase.exception.LiquibaseException: liquibase.exception.MigrationFailedException: Migration failed for changeset liquibase/changesets/02072023_create_authUser.yml::4::developer:
     Reason: liquibase.exception.DatabaseException: ERROR: type "long" does not exist
  Позиция: 35 [Failed SQL: (0) CREATE TABLE public.auth_user (id LONG GENERATED BY DEFAULT AS IDENTITY NOT NULL, username VARCHAR(255), password VARCHAR(255), role_name VARCHAR(255), CONSTRAINT auth_user_pkey PRIMARY KEY (id))]
	at liquibase.changelog.ChangeLogIterator.run(ChangeLogIterator.java:136)
	at liquibase.Liquibase.lambda$null$0(Liquibase.java:297)
	at liquibase.Scope.lambda$child$0(Scope.java:187)
	at liquibase.Scope.child(Scope.java:196)
	at liquibase.Scope.child(Scope.java:186)
	at liquibase.Scope.child(Scope.java:165)
	at liquibase.Scope.child(Scope.java:256)
	at liquibase.Liquibase.lambda$update$1(Liquibase.java:295)
	at liquibase.Scope.lambda$child$0(Scope.java:187)
	at liquibase.Scope.child(Scope.java:196)
	at liquibase.Scope.child(Scope.java:186)
	at liquibase.Scope.child(Scope.java:165)
	at liquibase.Liquibase.runInScope(Liquibase.java:2639)
	at liquibase.Liquibase.update(Liquibase.java:236)
	at liquibase.Liquibase.update(Liquibase.java:221)
	at liquibase.integration.spring.SpringLiquibase.performUpdate(SpringLiquibase.java:329)
	at liquibase.integration.spring.SpringLiquibase.afterPropertiesSet(SpringLiquibase.java:284)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1816)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1766)
	... 18 common frames omitted
Caused by: liquibase.exception.MigrationFailedException: Migration failed for changeset liquibase/changesets/02072023_create_authUser.yml::4::developer:
     Reason: liquibase.exception.DatabaseException: ERROR: type "long" does not exist
  Позиция: 35 [Failed SQL: (0) CREATE TABLE public.auth_user (id LONG GENERATED BY DEFAULT AS IDENTITY NOT NULL, username VARCHAR(255), password VARCHAR(255), role_name VARCHAR(255), CONSTRAINT auth_user_pkey PRIMARY KEY (id))]
	at liquibase.changelog.ChangeSet.execute(ChangeSet.java:744)
	at liquibase.changelog.visitor.UpdateVisitor.visit(UpdateVisitor.java:59)
	at liquibase.changelog.ChangeLogIterator$2.lambda$null$0(ChangeLogIterator.java:124)
	at liquibase.Scope.lambda$child$0(Scope.java:187)
	at liquibase.Scope.child(Scope.java:196)
	at liquibase.Scope.child(Scope.java:186)
	at liquibase.Scope.child(Scope.java:165)
	at liquibase.changelog.ChangeLogIterator$2.lambda$run$1(ChangeLogIterator.java:124)
	at liquibase.Scope.lambda$child$0(Scope.java:187)
	at liquibase.Scope.child(Scope.java:196)
	at liquibase.Scope.child(Scope.java:186)
	at liquibase.Scope.child(Scope.java:165)
	at liquibase.Scope.child(Scope.java:256)
	at liquibase.changelog.ChangeLogIterator$2.run(ChangeLogIterator.java:105)
	at liquibase.Scope.lambda$child$0(Scope.java:187)
	at liquibase.Scope.child(Scope.java:196)
	at liquibase.Scope.child(Scope.java:186)
	at liquibase.Scope.child(Scope.java:165)
	at liquibase.Scope.child(Scope.java:256)
	at liquibase.Scope.child(Scope.java:260)
	at liquibase.changelog.ChangeLogIterator.run(ChangeLogIterator.java:75)
	... 36 common frames omitted
Caused by: liquibase.exception.DatabaseException: ERROR: type "long" does not exist
  Позиция: 35 [Failed SQL: (0) CREATE TABLE public.auth_user (id LONG GENERATED BY DEFAULT AS IDENTITY NOT NULL, username VARCHAR(255), password VARCHAR(255), role_name VARCHAR(255), CONSTRAINT auth_user_pkey PRIMARY KEY (id))]
	at liquibase.executor.jvm.JdbcExecutor$ExecuteStatementCallback.doInStatement(JdbcExecutor.java:445)
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:77)
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:160)
	at liquibase.database.AbstractJdbcDatabase.execute(AbstractJdbcDatabase.java:1270)
	at liquibase.database.AbstractJdbcDatabase.executeStatements(AbstractJdbcDatabase.java:1252)
	at liquibase.changelog.ChangeSet.execute(ChangeSet.java:702)
	... 56 common frames omitted
Caused by: org.postgresql.util.PSQLException: ERROR: type "long" does not exist
  Позиция: 35
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2365)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:355)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:490)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:408)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:329)
	at org.postgresql.jdbc.PgStatement.executeCachedSql(PgStatement.java:315)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:291)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:286)
	at com.zaxxer.hikari.pool.ProxyStatement.execute(ProxyStatement.java:94)
	at com.zaxxer.hikari.pool.HikariProxyStatement.execute(HikariProxyStatement.java)
	at liquibase.executor.jvm.JdbcExecutor$ExecuteStatementCallback.doInStatement(JdbcExecutor.java:441)
	... 61 common frames omitted
08:16:12.560 [http-nio-8080-exec-4] ERROR r.s.l.s.w.service.EmployeeService - Неверный запрос пользователя
08:18:07.135 [http-nio-8080-exec-10] ERROR r.s.l.s.w.service.EmployeeService - There is no employee in file org.springframework.web.multipart.support.StandardMultipartHttpServletRequest$StandardMultipartFile@68d64542 
08:18:07.187 [http-nio-8080-exec-10] ERROR r.s.l.s.w.service.EmployeeService - Неверный запрос пользователя
08:19:28.007 [http-nio-8080-exec-2] ERROR r.s.l.s.w.service.EmployeeService - There is no report by id 1 
13:59:04.559 [main] ERROR liquibase.changelog - ChangeSet liquibase/changesets/02072023_create_employee1.yml::1::developer encountered an exception.
13:59:04.693 [main] ERROR o.s.boot.SpringApplication - Application run failed
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'liquibase' defined in class path resource [org/springframework/boot/autoconfigure/liquibase/LiquibaseAutoConfiguration$LiquibaseConfiguration.class]: liquibase.exception.MigrationFailedException: Migration failed for changeset liquibase/changesets/02072023_create_employee1.yml::1::developer:
     Reason: liquibase.exception.DatabaseException: ERROR: relation "employee1" already exists [Failed SQL: (0) CREATE TABLE public.employee1 (id INTEGER GENERATED BY DEFAULT AS IDENTITY NOT NULL, name VARCHAR(255), salary INTEGER NOT NULL, position_id INTEGER, CONSTRAINT employee1_pkey PRIMARY KEY (id))]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1770)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:598)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:520)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:326)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:324)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:200)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:313)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:200)
	at org.springframework.context.support.AbstractApplicationContext.getBean(AbstractApplicationContext.java:1156)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:931)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:608)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:146)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:733)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:435)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:311)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1305)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1294)
	at ru.skypro.lessons.springboot.weblibraryhw.WebLibraryHwApplication.main(WebLibraryHwApplication.java:10)
Caused by: liquibase.exception.LiquibaseException: liquibase.exception.MigrationFailedException: Migration failed for changeset liquibase/changesets/02072023_create_employee1.yml::1::developer:
     Reason: liquibase.exception.DatabaseException: ERROR: relation "employee1" already exists [Failed SQL: (0) CREATE TABLE public.employee1 (id INTEGER GENERATED BY DEFAULT AS IDENTITY NOT NULL, name VARCHAR(255), salary INTEGER NOT NULL, position_id INTEGER, CONSTRAINT employee1_pkey PRIMARY KEY (id))]
	at liquibase.changelog.ChangeLogIterator.run(ChangeLogIterator.java:136)
	at liquibase.Liquibase.lambda$null$0(Liquibase.java:297)
	at liquibase.Scope.lambda$child$0(Scope.java:187)
	at liquibase.Scope.child(Scope.java:196)
	at liquibase.Scope.child(Scope.java:186)
	at liquibase.Scope.child(Scope.java:165)
	at liquibase.Scope.child(Scope.java:256)
	at liquibase.Liquibase.lambda$update$1(Liquibase.java:295)
	at liquibase.Scope.lambda$child$0(Scope.java:187)
	at liquibase.Scope.child(Scope.java:196)
	at liquibase.Scope.child(Scope.java:186)
	at liquibase.Scope.child(Scope.java:165)
	at liquibase.Liquibase.runInScope(Liquibase.java:2639)
	at liquibase.Liquibase.update(Liquibase.java:236)
	at liquibase.Liquibase.update(Liquibase.java:221)
	at liquibase.integration.spring.SpringLiquibase.performUpdate(SpringLiquibase.java:329)
	at liquibase.integration.spring.SpringLiquibase.afterPropertiesSet(SpringLiquibase.java:284)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1816)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1766)
	... 18 common frames omitted
Caused by: liquibase.exception.MigrationFailedException: Migration failed for changeset liquibase/changesets/02072023_create_employee1.yml::1::developer:
     Reason: liquibase.exception.DatabaseException: ERROR: relation "employee1" already exists [Failed SQL: (0) CREATE TABLE public.employee1 (id INTEGER GENERATED BY DEFAULT AS IDENTITY NOT NULL, name VARCHAR(255), salary INTEGER NOT NULL, position_id INTEGER, CONSTRAINT employee1_pkey PRIMARY KEY (id))]
	at liquibase.changelog.ChangeSet.execute(ChangeSet.java:744)
	at liquibase.changelog.visitor.UpdateVisitor.visit(UpdateVisitor.java:59)
	at liquibase.changelog.ChangeLogIterator$2.lambda$null$0(ChangeLogIterator.java:124)
	at liquibase.Scope.lambda$child$0(Scope.java:187)
	at liquibase.Scope.child(Scope.java:196)
	at liquibase.Scope.child(Scope.java:186)
	at liquibase.Scope.child(Scope.java:165)
	at liquibase.changelog.ChangeLogIterator$2.lambda$run$1(ChangeLogIterator.java:124)
	at liquibase.Scope.lambda$child$0(Scope.java:187)
	at liquibase.Scope.child(Scope.java:196)
	at liquibase.Scope.child(Scope.java:186)
	at liquibase.Scope.child(Scope.java:165)
	at liquibase.Scope.child(Scope.java:256)
	at liquibase.changelog.ChangeLogIterator$2.run(ChangeLogIterator.java:105)
	at liquibase.Scope.lambda$child$0(Scope.java:187)
	at liquibase.Scope.child(Scope.java:196)
	at liquibase.Scope.child(Scope.java:186)
	at liquibase.Scope.child(Scope.java:165)
	at liquibase.Scope.child(Scope.java:256)
	at liquibase.Scope.child(Scope.java:260)
	at liquibase.changelog.ChangeLogIterator.run(ChangeLogIterator.java:75)
	... 36 common frames omitted
Caused by: liquibase.exception.DatabaseException: ERROR: relation "employee1" already exists [Failed SQL: (0) CREATE TABLE public.employee1 (id INTEGER GENERATED BY DEFAULT AS IDENTITY NOT NULL, name VARCHAR(255), salary INTEGER NOT NULL, position_id INTEGER, CONSTRAINT employee1_pkey PRIMARY KEY (id))]
	at liquibase.executor.jvm.JdbcExecutor$ExecuteStatementCallback.doInStatement(JdbcExecutor.java:445)
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:77)
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:160)
	at liquibase.database.AbstractJdbcDatabase.execute(AbstractJdbcDatabase.java:1270)
	at liquibase.database.AbstractJdbcDatabase.executeStatements(AbstractJdbcDatabase.java:1252)
	at liquibase.changelog.ChangeSet.execute(ChangeSet.java:702)
	... 56 common frames omitted
Caused by: org.postgresql.util.PSQLException: ERROR: relation "employee1" already exists
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2365)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:355)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:490)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:408)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:329)
	at org.postgresql.jdbc.PgStatement.executeCachedSql(PgStatement.java:315)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:291)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:286)
	at com.zaxxer.hikari.pool.ProxyStatement.execute(ProxyStatement.java:94)
	at com.zaxxer.hikari.pool.HikariProxyStatement.execute(HikariProxyStatement.java)
	at liquibase.executor.jvm.JdbcExecutor$ExecuteStatementCallback.doInStatement(JdbcExecutor.java:441)
	... 61 common frames omitted
14:03:16.206 [main] ERROR liquibase.changelog - ChangeSet liquibase/changesets/02072023_create_position.yml::2::developer encountered an exception.
14:03:16.599 [main] ERROR o.s.boot.SpringApplication - Application run failed
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'liquibase' defined in class path resource [org/springframework/boot/autoconfigure/liquibase/LiquibaseAutoConfiguration$LiquibaseConfiguration.class]: liquibase.exception.MigrationFailedException: Migration failed for changeset liquibase/changesets/02072023_create_position.yml::2::developer:
     Reason: liquibase.exception.DatabaseException: ERROR: relation "position" already exists [Failed SQL: (0) CREATE TABLE public.position (id INTEGER GENERATED BY DEFAULT AS IDENTITY NOT NULL, name VARCHAR(255), CONSTRAINT position_pkey PRIMARY KEY (id))]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1770)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:598)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:520)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:326)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:324)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:200)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:313)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:200)
	at org.springframework.context.support.AbstractApplicationContext.getBean(AbstractApplicationContext.java:1156)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:931)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:608)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:146)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:733)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:435)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:311)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1305)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1294)
	at ru.skypro.lessons.springboot.weblibraryhw.WebLibraryHwApplication.main(WebLibraryHwApplication.java:10)
Caused by: liquibase.exception.LiquibaseException: liquibase.exception.MigrationFailedException: Migration failed for changeset liquibase/changesets/02072023_create_position.yml::2::developer:
     Reason: liquibase.exception.DatabaseException: ERROR: relation "position" already exists [Failed SQL: (0) CREATE TABLE public.position (id INTEGER GENERATED BY DEFAULT AS IDENTITY NOT NULL, name VARCHAR(255), CONSTRAINT position_pkey PRIMARY KEY (id))]
	at liquibase.changelog.ChangeLogIterator.run(ChangeLogIterator.java:136)
	at liquibase.Liquibase.lambda$null$0(Liquibase.java:297)
	at liquibase.Scope.lambda$child$0(Scope.java:187)
	at liquibase.Scope.child(Scope.java:196)
	at liquibase.Scope.child(Scope.java:186)
	at liquibase.Scope.child(Scope.java:165)
	at liquibase.Scope.child(Scope.java:256)
	at liquibase.Liquibase.lambda$update$1(Liquibase.java:295)
	at liquibase.Scope.lambda$child$0(Scope.java:187)
	at liquibase.Scope.child(Scope.java:196)
	at liquibase.Scope.child(Scope.java:186)
	at liquibase.Scope.child(Scope.java:165)
	at liquibase.Liquibase.runInScope(Liquibase.java:2639)
	at liquibase.Liquibase.update(Liquibase.java:236)
	at liquibase.Liquibase.update(Liquibase.java:221)
	at liquibase.integration.spring.SpringLiquibase.performUpdate(SpringLiquibase.java:329)
	at liquibase.integration.spring.SpringLiquibase.afterPropertiesSet(SpringLiquibase.java:284)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1816)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1766)
	... 18 common frames omitted
Caused by: liquibase.exception.MigrationFailedException: Migration failed for changeset liquibase/changesets/02072023_create_position.yml::2::developer:
     Reason: liquibase.exception.DatabaseException: ERROR: relation "position" already exists [Failed SQL: (0) CREATE TABLE public.position (id INTEGER GENERATED BY DEFAULT AS IDENTITY NOT NULL, name VARCHAR(255), CONSTRAINT position_pkey PRIMARY KEY (id))]
	at liquibase.changelog.ChangeSet.execute(ChangeSet.java:744)
	at liquibase.changelog.visitor.UpdateVisitor.visit(UpdateVisitor.java:59)
	at liquibase.changelog.ChangeLogIterator$2.lambda$null$0(ChangeLogIterator.java:124)
	at liquibase.Scope.lambda$child$0(Scope.java:187)
	at liquibase.Scope.child(Scope.java:196)
	at liquibase.Scope.child(Scope.java:186)
	at liquibase.Scope.child(Scope.java:165)
	at liquibase.changelog.ChangeLogIterator$2.lambda$run$1(ChangeLogIterator.java:124)
	at liquibase.Scope.lambda$child$0(Scope.java:187)
	at liquibase.Scope.child(Scope.java:196)
	at liquibase.Scope.child(Scope.java:186)
	at liquibase.Scope.child(Scope.java:165)
	at liquibase.Scope.child(Scope.java:256)
	at liquibase.changelog.ChangeLogIterator$2.run(ChangeLogIterator.java:105)
	at liquibase.Scope.lambda$child$0(Scope.java:187)
	at liquibase.Scope.child(Scope.java:196)
	at liquibase.Scope.child(Scope.java:186)
	at liquibase.Scope.child(Scope.java:165)
	at liquibase.Scope.child(Scope.java:256)
	at liquibase.Scope.child(Scope.java:260)
	at liquibase.changelog.ChangeLogIterator.run(ChangeLogIterator.java:75)
	... 36 common frames omitted
Caused by: liquibase.exception.DatabaseException: ERROR: relation "position" already exists [Failed SQL: (0) CREATE TABLE public.position (id INTEGER GENERATED BY DEFAULT AS IDENTITY NOT NULL, name VARCHAR(255), CONSTRAINT position_pkey PRIMARY KEY (id))]
	at liquibase.executor.jvm.JdbcExecutor$ExecuteStatementCallback.doInStatement(JdbcExecutor.java:445)
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:77)
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:160)
	at liquibase.database.AbstractJdbcDatabase.execute(AbstractJdbcDatabase.java:1270)
	at liquibase.database.AbstractJdbcDatabase.executeStatements(AbstractJdbcDatabase.java:1252)
	at liquibase.changelog.ChangeSet.execute(ChangeSet.java:702)
	... 56 common frames omitted
Caused by: org.postgresql.util.PSQLException: ERROR: relation "position" already exists
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2365)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:355)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:490)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:408)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:329)
	at org.postgresql.jdbc.PgStatement.executeCachedSql(PgStatement.java:315)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:291)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:286)
	at com.zaxxer.hikari.pool.ProxyStatement.execute(ProxyStatement.java:94)
	at com.zaxxer.hikari.pool.HikariProxyStatement.execute(HikariProxyStatement.java)
	at liquibase.executor.jvm.JdbcExecutor$ExecuteStatementCallback.doInStatement(JdbcExecutor.java:441)
	... 61 common frames omitted
14:04:02.921 [main] ERROR liquibase.changelog - ChangeSet liquibase/changesets/02072023_create_report.yml::3::developer encountered an exception.
14:04:03.040 [main] ERROR o.s.boot.SpringApplication - Application run failed
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'liquibase' defined in class path resource [org/springframework/boot/autoconfigure/liquibase/LiquibaseAutoConfiguration$LiquibaseConfiguration.class]: liquibase.exception.MigrationFailedException: Migration failed for changeset liquibase/changesets/02072023_create_report.yml::3::developer:
     Reason: liquibase.exception.DatabaseException: ERROR: relation "report" already exists [Failed SQL: (0) CREATE TABLE public.report (id INTEGER GENERATED BY DEFAULT AS IDENTITY NOT NULL, file_path TEXT, CONSTRAINT report_pkey PRIMARY KEY (id))]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1770)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:598)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:520)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:326)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:324)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:200)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:313)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:200)
	at org.springframework.context.support.AbstractApplicationContext.getBean(AbstractApplicationContext.java:1156)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:931)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:608)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:146)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:733)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:435)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:311)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1305)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1294)
	at ru.skypro.lessons.springboot.weblibraryhw.WebLibraryHwApplication.main(WebLibraryHwApplication.java:10)
Caused by: liquibase.exception.LiquibaseException: liquibase.exception.MigrationFailedException: Migration failed for changeset liquibase/changesets/02072023_create_report.yml::3::developer:
     Reason: liquibase.exception.DatabaseException: ERROR: relation "report" already exists [Failed SQL: (0) CREATE TABLE public.report (id INTEGER GENERATED BY DEFAULT AS IDENTITY NOT NULL, file_path TEXT, CONSTRAINT report_pkey PRIMARY KEY (id))]
	at liquibase.changelog.ChangeLogIterator.run(ChangeLogIterator.java:136)
	at liquibase.Liquibase.lambda$null$0(Liquibase.java:297)
	at liquibase.Scope.lambda$child$0(Scope.java:187)
	at liquibase.Scope.child(Scope.java:196)
	at liquibase.Scope.child(Scope.java:186)
	at liquibase.Scope.child(Scope.java:165)
	at liquibase.Scope.child(Scope.java:256)
	at liquibase.Liquibase.lambda$update$1(Liquibase.java:295)
	at liquibase.Scope.lambda$child$0(Scope.java:187)
	at liquibase.Scope.child(Scope.java:196)
	at liquibase.Scope.child(Scope.java:186)
	at liquibase.Scope.child(Scope.java:165)
	at liquibase.Liquibase.runInScope(Liquibase.java:2639)
	at liquibase.Liquibase.update(Liquibase.java:236)
	at liquibase.Liquibase.update(Liquibase.java:221)
	at liquibase.integration.spring.SpringLiquibase.performUpdate(SpringLiquibase.java:329)
	at liquibase.integration.spring.SpringLiquibase.afterPropertiesSet(SpringLiquibase.java:284)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1816)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1766)
	... 18 common frames omitted
Caused by: liquibase.exception.MigrationFailedException: Migration failed for changeset liquibase/changesets/02072023_create_report.yml::3::developer:
     Reason: liquibase.exception.DatabaseException: ERROR: relation "report" already exists [Failed SQL: (0) CREATE TABLE public.report (id INTEGER GENERATED BY DEFAULT AS IDENTITY NOT NULL, file_path TEXT, CONSTRAINT report_pkey PRIMARY KEY (id))]
	at liquibase.changelog.ChangeSet.execute(ChangeSet.java:744)
	at liquibase.changelog.visitor.UpdateVisitor.visit(UpdateVisitor.java:59)
	at liquibase.changelog.ChangeLogIterator$2.lambda$null$0(ChangeLogIterator.java:124)
	at liquibase.Scope.lambda$child$0(Scope.java:187)
	at liquibase.Scope.child(Scope.java:196)
	at liquibase.Scope.child(Scope.java:186)
	at liquibase.Scope.child(Scope.java:165)
	at liquibase.changelog.ChangeLogIterator$2.lambda$run$1(ChangeLogIterator.java:124)
	at liquibase.Scope.lambda$child$0(Scope.java:187)
	at liquibase.Scope.child(Scope.java:196)
	at liquibase.Scope.child(Scope.java:186)
	at liquibase.Scope.child(Scope.java:165)
	at liquibase.Scope.child(Scope.java:256)
	at liquibase.changelog.ChangeLogIterator$2.run(ChangeLogIterator.java:105)
	at liquibase.Scope.lambda$child$0(Scope.java:187)
	at liquibase.Scope.child(Scope.java:196)
	at liquibase.Scope.child(Scope.java:186)
	at liquibase.Scope.child(Scope.java:165)
	at liquibase.Scope.child(Scope.java:256)
	at liquibase.Scope.child(Scope.java:260)
	at liquibase.changelog.ChangeLogIterator.run(ChangeLogIterator.java:75)
	... 36 common frames omitted
Caused by: liquibase.exception.DatabaseException: ERROR: relation "report" already exists [Failed SQL: (0) CREATE TABLE public.report (id INTEGER GENERATED BY DEFAULT AS IDENTITY NOT NULL, file_path TEXT, CONSTRAINT report_pkey PRIMARY KEY (id))]
	at liquibase.executor.jvm.JdbcExecutor$ExecuteStatementCallback.doInStatement(JdbcExecutor.java:445)
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:77)
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:160)
	at liquibase.database.AbstractJdbcDatabase.execute(AbstractJdbcDatabase.java:1270)
	at liquibase.database.AbstractJdbcDatabase.executeStatements(AbstractJdbcDatabase.java:1252)
	at liquibase.changelog.ChangeSet.execute(ChangeSet.java:702)
	... 56 common frames omitted
Caused by: org.postgresql.util.PSQLException: ERROR: relation "report" already exists
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2365)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:355)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:490)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:408)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:329)
	at org.postgresql.jdbc.PgStatement.executeCachedSql(PgStatement.java:315)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:291)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:286)
	at com.zaxxer.hikari.pool.ProxyStatement.execute(ProxyStatement.java:94)
	at com.zaxxer.hikari.pool.HikariProxyStatement.execute(HikariProxyStatement.java)
	at liquibase.executor.jvm.JdbcExecutor$ExecuteStatementCallback.doInStatement(JdbcExecutor.java:441)
	... 61 common frames omitted
14:04:42.798 [main] ERROR liquibase.changelog - ChangeSet liquibase/changesets/02072023_create_authUser.yml::4::developer encountered an exception.
14:04:42.947 [main] ERROR o.s.boot.SpringApplication - Application run failed
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'liquibase' defined in class path resource [org/springframework/boot/autoconfigure/liquibase/LiquibaseAutoConfiguration$LiquibaseConfiguration.class]: liquibase.exception.MigrationFailedException: Migration failed for changeset liquibase/changesets/02072023_create_authUser.yml::4::developer:
     Reason: liquibase.exception.DatabaseException: ERROR: relation "auth_user" already exists [Failed SQL: (0) CREATE TABLE public.auth_user (id BIGINT GENERATED BY DEFAULT AS IDENTITY NOT NULL, username VARCHAR(255), password VARCHAR(255), role_name VARCHAR(255), CONSTRAINT auth_user_pkey PRIMARY KEY (id))]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1770)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:598)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:520)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:326)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:324)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:200)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:313)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:200)
	at org.springframework.context.support.AbstractApplicationContext.getBean(AbstractApplicationContext.java:1156)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:931)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:608)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:146)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:733)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:435)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:311)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1305)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1294)
	at ru.skypro.lessons.springboot.weblibraryhw.WebLibraryHwApplication.main(WebLibraryHwApplication.java:10)
Caused by: liquibase.exception.LiquibaseException: liquibase.exception.MigrationFailedException: Migration failed for changeset liquibase/changesets/02072023_create_authUser.yml::4::developer:
     Reason: liquibase.exception.DatabaseException: ERROR: relation "auth_user" already exists [Failed SQL: (0) CREATE TABLE public.auth_user (id BIGINT GENERATED BY DEFAULT AS IDENTITY NOT NULL, username VARCHAR(255), password VARCHAR(255), role_name VARCHAR(255), CONSTRAINT auth_user_pkey PRIMARY KEY (id))]
	at liquibase.changelog.ChangeLogIterator.run(ChangeLogIterator.java:136)
	at liquibase.Liquibase.lambda$null$0(Liquibase.java:297)
	at liquibase.Scope.lambda$child$0(Scope.java:187)
	at liquibase.Scope.child(Scope.java:196)
	at liquibase.Scope.child(Scope.java:186)
	at liquibase.Scope.child(Scope.java:165)
	at liquibase.Scope.child(Scope.java:256)
	at liquibase.Liquibase.lambda$update$1(Liquibase.java:295)
	at liquibase.Scope.lambda$child$0(Scope.java:187)
	at liquibase.Scope.child(Scope.java:196)
	at liquibase.Scope.child(Scope.java:186)
	at liquibase.Scope.child(Scope.java:165)
	at liquibase.Liquibase.runInScope(Liquibase.java:2639)
	at liquibase.Liquibase.update(Liquibase.java:236)
	at liquibase.Liquibase.update(Liquibase.java:221)
	at liquibase.integration.spring.SpringLiquibase.performUpdate(SpringLiquibase.java:329)
	at liquibase.integration.spring.SpringLiquibase.afterPropertiesSet(SpringLiquibase.java:284)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1816)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1766)
	... 18 common frames omitted
Caused by: liquibase.exception.MigrationFailedException: Migration failed for changeset liquibase/changesets/02072023_create_authUser.yml::4::developer:
     Reason: liquibase.exception.DatabaseException: ERROR: relation "auth_user" already exists [Failed SQL: (0) CREATE TABLE public.auth_user (id BIGINT GENERATED BY DEFAULT AS IDENTITY NOT NULL, username VARCHAR(255), password VARCHAR(255), role_name VARCHAR(255), CONSTRAINT auth_user_pkey PRIMARY KEY (id))]
	at liquibase.changelog.ChangeSet.execute(ChangeSet.java:744)
	at liquibase.changelog.visitor.UpdateVisitor.visit(UpdateVisitor.java:59)
	at liquibase.changelog.ChangeLogIterator$2.lambda$null$0(ChangeLogIterator.java:124)
	at liquibase.Scope.lambda$child$0(Scope.java:187)
	at liquibase.Scope.child(Scope.java:196)
	at liquibase.Scope.child(Scope.java:186)
	at liquibase.Scope.child(Scope.java:165)
	at liquibase.changelog.ChangeLogIterator$2.lambda$run$1(ChangeLogIterator.java:124)
	at liquibase.Scope.lambda$child$0(Scope.java:187)
	at liquibase.Scope.child(Scope.java:196)
	at liquibase.Scope.child(Scope.java:186)
	at liquibase.Scope.child(Scope.java:165)
	at liquibase.Scope.child(Scope.java:256)
	at liquibase.changelog.ChangeLogIterator$2.run(ChangeLogIterator.java:105)
	at liquibase.Scope.lambda$child$0(Scope.java:187)
	at liquibase.Scope.child(Scope.java:196)
	at liquibase.Scope.child(Scope.java:186)
	at liquibase.Scope.child(Scope.java:165)
	at liquibase.Scope.child(Scope.java:256)
	at liquibase.Scope.child(Scope.java:260)
	at liquibase.changelog.ChangeLogIterator.run(ChangeLogIterator.java:75)
	... 36 common frames omitted
Caused by: liquibase.exception.DatabaseException: ERROR: relation "auth_user" already exists [Failed SQL: (0) CREATE TABLE public.auth_user (id BIGINT GENERATED BY DEFAULT AS IDENTITY NOT NULL, username VARCHAR(255), password VARCHAR(255), role_name VARCHAR(255), CONSTRAINT auth_user_pkey PRIMARY KEY (id))]
	at liquibase.executor.jvm.JdbcExecutor$ExecuteStatementCallback.doInStatement(JdbcExecutor.java:445)
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:77)
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:160)
	at liquibase.database.AbstractJdbcDatabase.execute(AbstractJdbcDatabase.java:1270)
	at liquibase.database.AbstractJdbcDatabase.executeStatements(AbstractJdbcDatabase.java:1252)
	at liquibase.changelog.ChangeSet.execute(ChangeSet.java:702)
	... 56 common frames omitted
Caused by: org.postgresql.util.PSQLException: ERROR: relation "auth_user" already exists
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2365)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:355)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:490)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:408)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:329)
	at org.postgresql.jdbc.PgStatement.executeCachedSql(PgStatement.java:315)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:291)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:286)
	at com.zaxxer.hikari.pool.ProxyStatement.execute(ProxyStatement.java:94)
	at com.zaxxer.hikari.pool.HikariProxyStatement.execute(HikariProxyStatement.java)
	at liquibase.executor.jvm.JdbcExecutor$ExecuteStatementCallback.doInStatement(JdbcExecutor.java:441)
	... 61 common frames omitted
14:07:54.396 [http-nio-8080-exec-7] ERROR o.s.s.w.a.UsernamePasswordAuthenticationFilter - An internal error occurred while trying to authenticate the user.
org.springframework.security.authentication.InternalAuthenticationServiceException: Cannot invoke "ru.skypro.lessons.springboot.weblibraryhw.security.AuthUser.getId()" because "authUser" is null
	at org.springframework.security.authentication.dao.DaoAuthenticationProvider.retrieveUser(DaoAuthenticationProvider.java:118)
	at org.springframework.security.authentication.dao.AbstractUserDetailsAuthenticationProvider.authenticate(AbstractUserDetailsAuthenticationProvider.java:133)
	at org.springframework.security.authentication.ProviderManager.authenticate(ProviderManager.java:182)
	at org.springframework.security.authentication.ProviderManager.authenticate(ProviderManager.java:201)
	at org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter.attemptAuthentication(UsernamePasswordAuthenticationFilter.java:85)
	at org.springframework.security.web.authentication.AbstractAuthenticationProcessingFilter.doFilter(AbstractAuthenticationProcessingFilter.java:231)
	at org.springframework.security.web.authentication.AbstractAuthenticationProcessingFilter.doFilter(AbstractAuthenticationProcessingFilter.java:221)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374)
	at org.springframework.security.web.authentication.logout.LogoutFilter.doFilter(LogoutFilter.java:107)
	at org.springframework.security.web.authentication.logout.LogoutFilter.doFilter(LogoutFilter.java:93)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374)
	at org.springframework.security.web.header.HeaderWriterFilter.doHeadersAfter(HeaderWriterFilter.java:90)
	at org.springframework.security.web.header.HeaderWriterFilter.doFilterInternal(HeaderWriterFilter.java:75)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374)
	at org.springframework.security.web.context.SecurityContextHolderFilter.doFilter(SecurityContextHolderFilter.java:82)
	at org.springframework.security.web.context.SecurityContextHolderFilter.doFilter(SecurityContextHolderFilter.java:69)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374)
	at org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter.doFilterInternal(WebAsyncManagerIntegrationFilter.java:62)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374)
	at org.springframework.security.web.session.DisableEncodeUrlFilter.doFilterInternal(DisableEncodeUrlFilter.java:42)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374)
	at org.springframework.security.web.FilterChainProxy.doFilterInternal(FilterChainProxy.java:233)
	at org.springframework.security.web.FilterChainProxy.doFilter(FilterChainProxy.java:191)
	at org.springframework.web.filter.DelegatingFilterProxy.invokeDelegate(DelegatingFilterProxy.java:352)
	at org.springframework.web.filter.DelegatingFilterProxy.doFilter(DelegatingFilterProxy.java:268)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:174)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:149)
	at org.springframework.web.filter.RequestContextFilter.doFilterInternal(RequestContextFilter.java:100)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:174)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:149)
	at org.springframework.web.filter.FormContentFilter.doFilterInternal(FormContentFilter.java:93)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:174)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:149)
	at org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:201)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:174)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:149)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:166)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:90)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:482)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:115)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:93)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:341)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:390)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:63)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:894)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1741)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:52)
	at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
	at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.NullPointerException: Cannot invoke "ru.skypro.lessons.springboot.weblibraryhw.security.AuthUser.getId()" because "authUser" is null
	at ru.skypro.lessons.springboot.weblibraryhw.security.AuthUserDTO.fromAuthUser(AuthUserDTO.java:16)
	at ru.skypro.lessons.springboot.weblibraryhw.security.SecurityUserDetailsService.loadUserByUsername(SecurityUserDetailsService.java:16)
	at org.springframework.security.authentication.dao.DaoAuthenticationProvider.retrieveUser(DaoAuthenticationProvider.java:103)
	... 57 common frames omitted
14:10:52.878 [http-nio-8080-exec-5] ERROR r.s.l.s.w.service.EmployeeService - Неверный запрос пользователя
14:11:03.145 [http-nio-8080-exec-8] ERROR r.s.l.s.w.service.EmployeeService - Неверный запрос пользователя
14:11:08.581 [http-nio-8080-exec-7] ERROR r.s.l.s.w.service.EmployeeService - Неверный запрос пользователя
14:27:00.295 [http-nio-8080-exec-2] ERROR r.s.l.s.w.service.EmployeeService - Неверный запрос пользователя
14:27:10.962 [http-nio-8080-exec-4] ERROR r.s.l.s.w.service.EmployeeService - Неверный запрос пользователя
17:20:39.942 [http-nio-8080-exec-3] ERROR o.h.e.jdbc.spi.SqlExceptionHelper - ERROR: duplicate key value violates unique constraint "employee1_pkey"
  Подробности: Key (id)=(1) already exists.
17:20:40.053 [http-nio-8080-exec-3] ERROR r.s.l.s.w.service.EmployeeService - Неверный запрос пользователя
17:23:19.982 [http-nio-8080-exec-6] ERROR r.s.l.s.w.service.EmployeeService - Неверный запрос пользователя
17:23:36.982 [http-nio-8080-exec-7] ERROR r.s.l.s.w.service.EmployeeService - Неверный запрос пользователя
17:25:12.154 [http-nio-8080-exec-10] ERROR r.s.l.s.w.service.EmployeeService - Неверный запрос пользователя
17:25:32.836 [http-nio-8080-exec-2] ERROR r.s.l.s.w.service.EmployeeService - Неверный запрос пользователя
17:25:39.295 [http-nio-8080-exec-1] ERROR r.s.l.s.w.service.EmployeeService - Неверный запрос пользователя
17:31:50.958 [http-nio-8080-exec-1] ERROR r.s.l.s.w.service.EmployeeService - Неверный запрос пользователя
17:34:46.090 [http-nio-8080-exec-4] ERROR r.s.l.s.w.service.EmployeeService - Неверный запрос пользователя
17:34:51.845 [http-nio-8080-exec-5] ERROR r.s.l.s.w.service.EmployeeService - Неверный запрос пользователя
18:50:19.948 [http-nio-8080-exec-6] ERROR r.s.l.s.w.service.EmployeeService - Неверный запрос пользователя
18:50:43.804 [http-nio-8080-exec-10] ERROR r.s.l.s.w.service.EmployeeService - Неверный запрос пользователя
18:51:16.083 [http-nio-8080-exec-1] ERROR r.s.l.s.w.service.EmployeeService - Неверный запрос пользователя
18:51:42.467 [http-nio-8080-exec-9] ERROR r.s.l.s.w.service.EmployeeService - Неверный запрос пользователя
18:52:34.880 [http-nio-8080-exec-4] ERROR r.s.l.s.w.service.EmployeeService - Неверный запрос пользователя
19:01:07.416 [http-nio-8080-exec-8] ERROR r.s.l.s.w.service.EmployeeService - Неверный запрос пользователя
19:22:19.789 [http-nio-8080-exec-10] ERROR r.s.l.s.w.service.EmployeeService - Неверный запрос пользователя
19:37:09.400 [http-nio-8080-exec-5] ERROR r.s.l.s.w.service.EmployeeService - Неверный запрос пользователя
19:59:31.915 [http-nio-8080-exec-6] ERROR r.s.l.s.w.service.EmployeeService - Неверный запрос пользователя
09:16:53.566 [http-nio-8080-exec-10] ERROR r.s.l.s.w.service.EmployeeService - Неверный запрос пользователя
09:17:25.627 [http-nio-8080-exec-1] ERROR r.s.l.s.w.service.EmployeeService - Неверный запрос пользователя
09:21:30.678 [http-nio-8080-exec-2] ERROR r.s.l.s.w.service.EmployeeService - Неверный запрос пользователя
09:27:18.177 [http-nio-8080-exec-10] ERROR r.s.l.s.w.service.EmployeeService - Неверный запрос пользователя
09:27:30.273 [http-nio-8080-exec-2] ERROR r.s.l.s.w.service.EmployeeService - Неверный запрос пользователя
09:28:26.481 [http-nio-8080-exec-1] ERROR r.s.l.s.w.service.EmployeeService - Неверный запрос пользователя
09:28:59.976 [http-nio-8080-exec-3] ERROR r.s.l.s.w.service.EmployeeService - Неверный запрос пользователя
09:29:28.535 [http-nio-8080-exec-6] ERROR r.s.l.s.w.service.EmployeeService - Неверный запрос пользователя
09:29:55.182 [http-nio-8080-exec-4] ERROR r.s.l.s.w.service.EmployeeService - Неверный запрос пользователя
09:50:51.867 [http-nio-8080-exec-9] ERROR r.s.l.s.w.service.EmployeeService - Неверный запрос пользователя
09:59:14.001 [http-nio-8080-exec-8] ERROR r.s.l.s.w.service.EmployeeService - Неверный запрос пользователя
10:04:47.554 [http-nio-8080-exec-6] ERROR r.s.l.s.w.service.EmployeeService - Неверный запрос пользователя
13:30:10.214 [http-nio-8080-exec-10] ERROR r.s.l.s.w.service.EmployeeService - There is no employee in file org.springframework.web.multipart.support.StandardMultipartHttpServletRequest$StandardMultipartFile@354bfbdc 
13:30:10.622 [http-nio-8080-exec-10] ERROR r.s.l.s.w.service.EmployeeService - Неверный запрос пользователя
19:49:55.109 [http-nio-8080-exec-9] ERROR r.s.l.s.w.service.EmployeeService - There is no report by id 1 
19:49:55.188 [http-nio-8080-exec-9] ERROR r.s.l.s.w.service.EmployeeService - Страница не найдена, проверьте адрес URL
19:50:12.981 [http-nio-8080-exec-8] ERROR r.s.l.s.w.service.EmployeeService - There is no report by id 1 
